repositories.yaml

```diff
@@ -805,45 +805,28 @@ applications:
     notification_emails:
       - vpn@mozilla.com
       - amarchesini@mozilla.com
+      - brizental@mozilla.com
```

Comment:
Adding Bea here because she was in the notification list for the Android app, but now that she has moved to a different role, maybe this should actually be removed altogether?

And/or should I add myself?  (I'm unsure what kind of notifications these would be)

Reply:
@brizental do you still want these notifications?

Reply:
Hey Sean, I am acutally working on the VPN team now so yeah, you can leave me there :D Thanks for asking!